redirect.php

```diff
@@ -19,8 +19,14 @@
         Tools::redirect('authentication.php?back=order.php');
     }
 
+    $cart = Context::getContext()->cart;
```

Comment:
Variable to use one time? A die message looks bad in this context, it can be changes for redirect (see line 19 in this file)

Reply:
Se usa dos veces para validar y para pasar al metodo redirect